workflows/internal.workflow.yml

```diff
@@ -192,3 +220,25 @@ jobs:
           MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
           APPSTORE_KEY_BASE64: ${{ secrets.APPSTORE_KEY_BASE64 }}
           environment: qa
+      # Flutter DEV build
+      - if: needs.config.outputs.DEV_ENV_SUPPORTED == 'true'
+        name: Flutter DEV Build
+        run: flutter build ios -t lib/${entry_file}.dart --flavor ${flavor} --release --no-codesign
+        env:
+          entry_file: main_dev.dart
```

Comment:
I think `entry_file` should be just `main_dev` without the `.dart` suffix, otherwise it will generate `lib/main_dev.dart.dart`